engine/engine.go

```diff
@@ -97,16 +105,20 @@ func (c *TofuEngine) Run(req *tgengine.RunRequest, stream tgengine.Engine_RunSer
 	// Stream stdout
 	go func() {
```

Comment:
And in order not to have to use err throughout the entire function to return errors, I would recommend using

```go
g, ctx := errgroup.WithContext(ctx)
g.Go(func() error {} ....)
if err := g.Wait(); err != nil ....)
```

Reply:
I haven't used this before, and I think it's changing the purpose of this PR, which is to make this repo track the linting in the Terragrunt repo. Would you mind if we deferred this work? If possible, I'd also like to avoid doing that update, as I don't really get the point.

Reply:
Of course. It's not critical.